README.md

```diff
@@ -407,6 +408,28 @@ Enabling stream adds matching topics. For instance, enabling the gyro and accel
 
 <hr>
 
+<h3 id="rgbd">
+  RGBD Topic
+</h3>
+
+RGBD new topic, publishing [RGB + Depth] in the same message (see RGBD.msg for reference). For now, works only with depth aligned to color images.

```

Comment:
I think it worth mentioning that the RGB + Depth are synchronized by frame time tag, right?

Reply:
done